demo/index.html

```diff
@@ -25,18 +26,28 @@
 			<h3>Basic <code>cosmoz-image-viewer</code>.</h3>
 			<demo-snippet>
 				<template>
-					<cosmoz-image-viewer></cosmoz-image-viewer>
+					<dom-bind class="scope">
+						<template is="dom-bind" class="scope">
+							<cosmoz-image-viewer images="[[images]]"></cosmoz-image-viewer>
```

Comment:
Two needed?

Reply:
To showcase the shared "detached" window.